share/steel/examples/pulse/lib/Pulse.Lib.Core.fsti

```diff
@@ -206,7 +194,29 @@ val bind_stt
   : stt b pre1 post2
 
 inline_for_extraction
-val lift_stt_atomic (#a:Type u#a) (#opens:inames) (#pre:vprop) (#post:a -> vprop)
+val lift_stt_atomic0
+  (#a:Type u#0)
+  (#opens:inames)
+  (#pre:vprop)
+  (#post:a -> vprop)
+  (e:stt_atomic a opens pre post)
+  : stt a pre post
+
+inline_for_extraction
+val lift_stt_atomic1
+  (#a:Type u#1)
+  (#opens:inames)
+  (#pre:vprop)
+  (#post:a -> vprop)
+  (e:stt_atomic a opens pre post)
+  : stt a pre post
+
+inline_for_extraction
+val lift_stt_atomic2
```

Comment:
Note, these specialized lifts for some common universes are a consequence of the universe bound on actions.